### PR TITLE
Pw 7281/3ds2 config

### PR DESCRIPTION
**Pull request by zenit2001**
## Summary

This pull request removes the custom 3DS setting in the configuration page.
Also, it replaces the deprecated **allow3DS2** parameter, with a new parameter called **[authenticationData](https://docs.adyen.com/api-explorer/Checkout/latest/post/payments#request-authenticationData)**.
Additionally, **adyen3ds2.isml** file is remove, since its not being used anymore.

### CODE REVIEW

File: src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js [hunk 1]
```diff
@@ -453,7 +453,10 @@ var adyenHelperObj = {
 
   // adds 3DS2 fields to an Adyen Checkout payments Request
   add3DS2Data(jsonObject) {
-    jsonObject.additionalData.allow3DS2 = true;
+    jsonObject.authenticationData = {};
```
Comment (wboereboom) [inline src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js:456]:
```suggestion
    jsonObject.authenticationData = {
      threeDSRequestData: {
        nativeThreeDS: "preferred",
      },
    }
```

File: src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js [hunk 2]
```diff
@@ -574,7 +577,7 @@ var adyenHelperObj = {
     order.custom.Adyen_value = '0';
     // Save full response to transaction custom attribute
     paymentInstrument.paymentTransaction.custom.Adyen_log =  JSON.stringify(result);
-
+
```
Comment (wboereboom) [inline src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js:580]:
```suggestion
```

**Review by wboereboom — COMMENTED** [review]
Are there more templates we should remove? such as `src/cartridges/int_adyen_controllers/cartridge/templates/default/adyenpaymentredirect.isml`. Will we include that in this PR as well?

**Review by wboereboom — COMMENTED** [review]
Are there more templates we should remove? such as `src/cartridges/int_adyen_controllers/cartridge/templates/default/adyenpaymentredirect.isml`. Will we include that in this PR as well?

**Review by zaiddreakh — APPROVED** [review]

**Review by maassenbas — APPROVED** [review]

**Pull request merged**